app/services/move_events/params_validator.rb

```diff
@@ -32,17 +32,21 @@ class ParamsValidator
       record.errors.add(attr, 'must be formatted as a valid ISO-8601 date-time')
     end
 
-    validates_with LocationValidator, locations: [:from_location], if: -> { type == 'lockouts' }
-    validates_with LocationValidator, locations: [:to_location], if: -> { type == 'redirects' }
+    validates_with LocationValidator, locations: [:from_location_id], if: -> { type == 'lockouts' }
+    validates_with LocationValidator, locations: [:to_location_id], if: -> { type == 'redirects' }
+    validates_with Moves::MoveTypeValidator, if: -> { type == 'redirects' }
 
-    def initialize(params)
+    delegate :move_type, :from_location, to: :move
```

Comment:
I think these names could potentially be confusing:
```
from_location --> move's from_location
from_location_id --> event's from_location
```

how about:
```
move_from_location --> move's from_location
event_from_location --> event's from_location(_id)
```
?

Reply:
Discussed offline, this isn't trivial to change as the `MoveTypeValidator` needs to believe that this is really a `Move` model with the same `move_type` and `from_location` attributes. Agreed to have another look at that in the next PR for the other upcoming move types. In the meantime this is enough to unblock front end work.